examples/pstore/pscli.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"fmt"
+	"storj.io/storj/pkg/pstore"
+	"github.com/urfave/cli"
+	"github.com/zeebo/errs"
+	"log"
+	"os"
+	"sort"
+)
+
+var ArgError = errs.Class("argError")
+
+func main() {
+	app := cli.NewApp()
+
+	app.Name = "Piece Store CLI"
+	app.Usage = "Store data in hash folder structure"
+	app.Version = "1.0.0"
+
+	app.Flags = []cli.Flag{}
+
+	app.Commands = []cli.Command{
+		{
+			Name:      "store",
+			Aliases:   []string{"s"},
+			Usage:     "Store data by hash",
+			ArgsUsage: "[hash] [dataPath] [storeDir]",
+			Action: func(c *cli.Context) error {
+				if c.Args().Get(0) == "" {
+					return ArgError.New("Missing data Hash")
+				}
+
+				if c.Args().Get(1) == "" {
+					return ArgError.New("No input file specified")
+				}
+
+				if c.Args().Get(2) == "" {
+					return ArgError.New("No output directory specified")
+				}
+
+				file, err := os.Open(c.Args().Get(1))
+				if err != nil {
+					return err
+				}
+				// Close the file when we are done
+				defer file.Close()
+
+				fileInfo, err := os.Stat(c.Args().Get(1))
+
+				if fileInfo.IsDir() {
```

Comment:
this won't happen since os.Open worked

Reply:
os.Open does not error on opening existing directories